content/modals/US/product_list.json

```
@@ -29,6 +29,52 @@
                 "viewName": "PAYPAL_CREDIT_NO_INTEREST"
             }
         ],
+        "tiles": {
+            "payLater": [
+                {
+                    "header": "Pay in 4",
+                    "icon": "pay-in-four-tile",
+                    "body": "Interest-free payments every 2 weeks, starting today.",
+                    "viewName": "PAY_LATER_SHORT_TERM"
+                },
+                {
+                    "header": "Pay monthly",
+                    "icon": "pay-monthly-tile",
+                    "body": "Split your purchase into equal monthly payments.",
+                    "viewName": "PAY_LATER_LONG_TERM"
+                }
+            ],
+            "credit": [
+                {
+                    "header": "PayPal Credit",
+                    "icon": "paypal-credit-tile",
+                    "body": "No Interest if paid in full in 6 months for purchases of $99+.",
+                    "viewName": "PAYPAL_CREDIT_NO_INTEREST"
+                }
+            ]
+        },
+        "payLaterTiles": [
+            {
+                "header": "Pay in 4",
+                "icon": "pay-in-four-tile",
+                "body": "Interest-free payments every 2 weeks, starting today.",
+                "viewName": "PAY_LATER_SHORT_TERM"
+            },
+            {
+                "header": "Pay monthly",
+                "icon": "pay-monthly-tile",
+                "body": "Split your purchase into equal monthly payments.",
+                "viewName": "PAY_LATER_LONG_TERM"
+            }
+        ],
+        "creditTiles": [
+            {
+                "header": "PayPal Credit",
+                "icon": "paypal-credit-tile",
+                "body": "No Interest if paid in full in 6 months for purchases of $99+.",
+                "viewName": "PAYPAL_CREDIT_NO_INTEREST"
+            }
+        ],
```

```suggestion
        "payLaterTiles": [
            {
                "header": "Pay in 4",
                "icon": "pay-in-four-tile",
                "body": "Interest-free payments every 2 weeks, starting today.",
                "viewName": "PAY_LATER_SHORT_TERM"
            },
            {
                "header": "Pay monthly",
                "icon": "pay-monthly-tile",
                "body": "Split your purchase into equal monthly payments.",
                "viewName": "PAY_LATER_LONG_TERM"
            }
        ],
        "creditTiles": [
            {
                "header": "PayPal Credit",
                "icon": "paypal-credit-tile",
                "body": "No Interest if paid in full in 6 months for purchases of $99+.",
                "viewName": "PAYPAL_CREDIT_NO_INTEREST"
            }
        ],
```
Can this be removed since `tiles` is being used?